diff --color -rupN linuxptp-4.0-orig/msg.h linuxptp-4.0/msg.h
--- linuxptp-4.0-orig/msg.h	2023-05-22 21:27:43
+++ linuxptp-4.0/msg.h	2023-07-10 12:26:16
@@ -32,7 +32,7 @@
 
 /* Version definition for IEEE 1588-2019 */
 #define PTP_MAJOR_VERSION	2
-#define PTP_MINOR_VERSION	1
+#define PTP_MINOR_VERSION	0
 #define PTP_VERSION		(PTP_MINOR_VERSION << 4 | PTP_MAJOR_VERSION)
 
 #define MAJOR_VERSION_MASK	0x0f
